docs/chapters/baseplatform/deploying-and-booting-the-image.rst

```diff
@@ -0,0 +1,64 @@
+Writing the image
+=================
+
+This section describes how to write the build output, a ready disk-image, to a removable media. Assuming you are positioned in the PELUX top-level Yocto directory follow the instructions stated below. Further information can be found in the `Poky documentation`_.
+
+NUC / Minnowboard
+-----------------
+
+There are three different methods of writing an Intel ``.hddimg`` or ``.wic`` image to a USB-stick.
+
+1. Use the ``dd`` utility to write the image to the raw block device. For example:
+
+.. code-block:: bash
+
+    dd if=build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg \
```

Comment:
Could you rewrite it a bit so you mention where to find it in the build directory but that it would be easier to use for someone who just downloaded it for example into their `~/Downloads` directory?